server/shared/types/terms/term.ts

```diff
@@ -166,6 +167,8 @@ export type BaseQ = {
 		/** Applies to categorical and condition terms */
 		| 'custom-groupset'
 		| 'custom-samplelst'
+
+	groupsetting?: GroupSetting // TODO XXX FIXME clean up all these mess!!!!
```

Comment:
would like Gav to clean up such mess @gavrielm @siosonel

Reply:
Ok will do